pkg/generate/app/imagestreamlookup.go

```diff
@@ -162,10 +162,14 @@ func (r ImageStreamSearcher) Search(precise bool, terms ...string) (ComponentMat
 							continue
 						}
 
-						// If the user specified a tag in their search string (followTag == false),
-						// then score this match lower.
-						tagScore := score
+						// at best this is a partial match situation.  The user didn't
+						// specify a tag, so we tried "latest" but could not find an image associated
+						// with the latest tag (or one that is followed by the latest tag), or
+						// they specified a tag that we could not find.
+						tagScore := score + 0.1
 						if !followTag {
+							// If the user specified a tag in their search string (followTag == false),
+							// then score this match lower.
 							tagScore += 0.5
```

Comment:
@jim-minter i have half a mind to just drop this if statement and always set tagScore:=score+0.5, i don't think there's a meaningful distinction between score=0.1 and score=0.6, nor am i convinced there's a good justification for why one should be 0.1 and the other should be 0.6, but let me know what you think.